pkg/bootstrap/config.go

```diff
@@ -50,6 +50,8 @@ const (
 	// required stats are used by readiness checks.
 	requiredEnvoyStatsMatcherInclusionPrefixes = "cluster_manager,listener_manager,server,cluster.xds-grpc,wasm"
 
+	rbacEnvoyStatsMatcherInclusionSuffix = "allowed,denied"
```

Comment:
This seems to be a pretty common words that probably will be used by other stats as well. In order to not accidently generates more stats, is it possible to make the suffix more specific so that only rbac stats is included?

Reply:
It is indeed used by other security filters (ext_authz and jwt). 

These other stats are actually very useful and should be included anyway, I can update this PR to `"rbac.allowed,rbac.denied,shadow_allowed,shadow_denied"` to only include RBAC stats and send out another PR to include those additional stats for other filters, but that will make this list very long and slightly worse performance.

Let me know which one you prefer and I will update.

Reply:
Let's make it specific. The fear is that one day some cluster stats are added with allowed/denied suffix and it gets exposed, which is not ideal.

Reply:
Updated.